apps/initializer/lib/initializer.ex

```diff
@@ -8,6 +8,8 @@ defmodule Initializer do
     quote location: :keep do
       use GenServer
       use Retry
```

Comment:
i think you can remove the reference to `use Retry` if you've got `use Annotated.Retry`

Reply:
This is still being used. Leaving it in for now.

Reply:
Converted this to the new Annotated.Retry the other place it was being used here. Take a look at my updated version when you get a chance.